src/util/getTopologicalOrder.ts

```diff
@@ -3,7 +3,11 @@ export interface Edge {
 	target: string;
 }
 
-// Uses the Khan Algorithm loosely
+/**
+	* Uses the Khan Algorithm
+	* @param {Set<Edge>} graph - A graph represented as set of edges
```

Comment:
In your usecase, is it possible, that you have Vertices that are isolated, i.e. that are not connected to other vertices?
If so, modelling a graph as a set of edges would not suffice

Reply:
The usecase is to import Files in the correct order so there will not be an isolated edge and if there were an isolated edge it wouldn't matter. Though I should add this to the description as I did with `getVertexesWithNoInEdge()` I should also correct the naming. There is no such thing as "Vertixes" oops.

I think I will call the nodes to make the names shorter.

Reply:
If you would want to keep it close to the formal notion of a graph, you could also go with Vertices, which would be the correct plural iirc.